src/webui/www/private/scripts/client.js

```diff
@@ -581,14 +585,23 @@ window.addEvent('load', function() {
                 }
                 clearTimeout(syncMainDataTimer);
                 syncMainDataTimer = syncMainData.delay(getSyncMainDataInterval());
+                requestInProgress = false;
+                $('syncing_div').set('html', '');
             }
-        }).send();
+        });
+        requestInProgress = true;
+        request.send();
     };
 
     updateMainData = function() {
         torrentsTable.updateTable();
-        clearTimeout(syncMainDataTimer);
-        syncMainDataTimer = syncMainData.delay(100);
+        if (!requestInProgress){
+            clearTimeout(syncMainDataTimer);
+            syncMainDataTimer = syncMainData.delay(100);
+        }
+        else {
+            $('syncing_div').set('html', 'Syncing...');
```

Comment:
You should make the "Syncing..." string translatable.

Reply:
Ok, done, I used the same context as the `error_div`. Is that all that is necessary?